lib/bundler/fetcher.rb

```diff
@@ -295,7 +296,13 @@ def bundler_cert_store
         end
       else
         store.set_default_paths
-        certs = File.expand_path("../ssl_certs/*/*.pem", __FILE__)
+
+        rubygems_certs_dir = File.expand_path("../../rubygems/ssl_certs", __FILE__)
+        unless File.exists?(rubygems_certs_dir)
+          rubygems_certs_dir = File.join(RbConfig::CONFIG["rubylibdir"], "rubygems", "ssl_certs")
+        end
```

Comment:
Rubygems has `Gem::RUBYGEMS_DIR` to inform of its own installation location:

https://github.com/rubygems/rubygems/blob/8b81173e0463fe362187319d4459c563b9b848c8/lib/rubygems.rb#L116that

Maybe it's a better fit here since it will always be correct and doesn't make us check for different possible installation structures?

Reply:
Thanks, I didn't know it. It always provides in user's environment. We should use it.

Reply:
There is even `Gem::Request.get_cert_files` available since RubyGems 2.3, so it should be even better ...

Reply:
I'm fine with using `Gem::Request.get_cert_files` :+1:.

Reply:
Thanks again. Update done.